### PR TITLE
LPS-145478 Add new friendly URL support to document display pages

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/layout/display/page/FileEntryLayoutDisplayPageProvider.java [hunk 1]
```diff
@@ -70,16 +74,33 @@ public String getClassName() {
...
+		if (friendlyURLEntry != null) {
+			return getLayoutDisplayPageObjectProvider(
+				new InfoItemReference(
+					FileEntry.class.getName(), friendlyURLEntry.getClassPK()));
+		}
+
+		if (Validator.isNumber(urlTitle)) {
+			return getLayoutDisplayPageObjectProvider(
+				new InfoItemReference(
+					FileEntry.class.getName(), GetterUtil.getLong(urlTitle)));
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/layout/display/page/FileEntryLayoutDisplayPageProvider.java:90]: I have a question, more ti know why and  get it clearer.

Why did you decided to use the getterUtil over the Long.valueOf here?

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**